src/runtime/types/index.ts

```diff
@@ -65,10 +73,14 @@ export interface MessagesLoader {
 }
 
 export interface ConfigureOptions {
-  fallbackLocale: string;
-  formats?: Partial<Formats>;
-  initialLocale?: string;
-  loadingDelay?: number;
-  warnOnMissingMessages?: boolean;
-  ignoreTag?: boolean;
+  fallbackLocale: string | null | undefined;
+  formats: Formats;
```

Comment:
The dev should be able to pass just a partial version of the formats object

Reply:
4.
Thank you, I confirmed that it is enough to be typed as `Optional<Formats>` anywhere.